codecept.conf.js

```diff
@@ -0,0 +1,55 @@
+exports.config = {
+  output: './output',
+  helpers: {
+    Playwright: {
+      browser: 'chromium',
+      url: 'https://www.saucedemo.com',
+      show: false,
+    },
+  },
+  include: {
+    I: './steps_file.js',
+    loginPagePage: './bdd_tests/pages/LoginPage.js',
+    authResultPage: './bdd_tests/pages/AuthResult.js',
+  },
+  mocha: {},
+  bootstrap: null,
+  timeout: null,
+  teardown: null,
+  hooks: [],
+  gherkin: {
+    features: './features/login.feature',
+    steps: ['./step_definitions/login_steps.js'],
+  },
+  plugins: {
+    screenshotOnFail: {
+      enabled: true,
+    },
+    tryTo: {
+      enabled: true,
+    },
+    retryFailedStep: {
+      enabled: true,
+    },
+    retryTo: {
+      enabled: true,
+    },
+    eachElement: {
+      enabled: true,
+    },
+    pauseOnFail: {},
+  },
+  stepTimeout: 0,
+  stepTimeoutOverride: [
+    {
+      pattern: 'wait.*',
+      timeout: 0,
+    },
+    {
+      pattern: 'amOnPage',
+      timeout: 0,
+    },
+  ],
+  tests: './bdd_tests/bdd_tests.js',
```

Comment:
```suggestion
  tests: './bdd_tests/**/*_test.js',
```

Чтобы потенциально много тестов можно было запустить, сейчас один файл ищет